### ISSUE TITLE
AK8963 accuracy in degrees

### ISSUE THREAD

**Issue by ghost** [kriswiner/MPU9250#115]
I calibrated the AK8963 magnetometer.

Then I rotated the AK8963 the 360 degrees. Here is X-Y view.

![1111](https://cloud.githubusercontent.com/assets/7097788/23089498/6fc46b62-f53e-11e6-8dd2-1768ff8bd5b5.png)

Using atan2 function with X and Y magnitudes of the magnetometer, I got only 10 degrees of accuracy.

Is this typical if you only use the magnetometer alone?

**Comment by kriswiner** [comment]
Yes, for heading. With all three sensors and even simple sensor fusion like open-source Madgwick or Mahony you can get +/- 1 degree accuracy in pitch or roll and +/- 4 degree accuracy in yaw or heading. EM7180 can do even better, at +/- 2 degrees in heading. See: https://github.com/kriswiner/MPU-6050/wiki/Hardware-Sensor-Fusion-Solutions
